src/instructlab/generator/generate_data.py

```diff
@@ -371,7 +357,6 @@ def generate_data(
     num_cpus: Optional[int] = None,
     num_instructions_to_generate: Optional[int] = None,
     num_prompt_instructions=2,
-    request_batch_size=5,
```

Comment:
does this simplification actually changed this to (and fixed it as) `request_batch_size=1` effectively?